packages/access-token/src/json-store.js

```diff
@@ -9,7 +9,7 @@ import Readlines from 'n-readlines';
 //
 // For compatibility, because it's tricky to diagnose that the file location has
 // changed when you're just trying to use the wallet from the browser.
-const DATA_FILE = 'swingset-kernel-state.jsonlines';
+export const DATA_FILE = 'swingset-kernel-state.jsonlines';
```

Comment:
I'd prefer not exporting this implementation details and instead have the locking mechanism implemented in the store itself.